File: codigos/if-elif-else.py
idade = int(input('Nos conte sua idade: '))
if idade < 13:
    print('Você é criança')
elif idade < 18:
    print('Você é adolescente')
elif idade < 60:
    print('Você é adulto')
else:
    print('Você é idoso')

File: codigos/if-elif-else.php
<?php
echo("Nos conte sua idade: ");
$idade = trim(fgets(STDIN));
if ($idade < 13) {
    echo("Você é criança\n");
} elseif ($idade <18) {
    echo("Você é adolescente\n");
} elseif ($idade < 60) {
    echo("Você é adulto\n");
} else {
    echo("Você é idoso\n");
}
